themes/courses.labster.com/lms/templates/registration/coach_create_password_reset_email.html

```diff
@@ -1,15 +1,10 @@
 {% load i18n %}{% load url from future %}{% autoescape off %}
-{% blocktrans %}You're receiving this e-mail because you requested a password reset for your user account at {{ site_name }}.{% endblocktrans %}
+{% blocktrans %}
+We have created a course for you and you are almost set. The last step is for you to set your password
+associated with the following login {{ email }}. Please go to to the following page :
+{% endblocktrans %}
+{% block reset_link %}{{ protocol }}://{{ site_name }}{% url 'student.views.password_reset_confirm_wrapper' uidb36=uid token=token %}{% endblock %}
 
-{% trans "Please go to the following page and choose a new password:" %}
-{% block reset_link %}
-{{ protocol }}://{{ site_name }}{% url 'student.views.password_reset_confirm_wrapper' uidb36=uid token=token %}
-{% endblock %}
-
-{% trans "If you didn't request this change, you can disregard this email - we have not yet reset your password." %}
-
-{% trans "Thanks for using our site!" %}
-
-{% blocktrans %}The {{ platform_name }} Team{% endblocktrans %}
+{% trans "If you have any questions, please contact your Customer Success Specialist or support@labster.com" %}
```

Comment:
Same here - can we use one of the settings constants?

Reply:
Same above